src/HttpClientInterception/HttpClientInterceptorOptions.cs

```diff
@@ -158,7 +161,8 @@ public HttpClientInterceptorOptions Deregister(HttpMethod method, Uri uri)
             string mediaType = JsonMediaType,
             IEnumerable<KeyValuePair<string, IEnumerable<string>>> responseHeaders = null,
             IEnumerable<KeyValuePair<string, IEnumerable<string>>> contentHeaders = null,
-            Func<HttpRequestMessage, Task> onIntercepted = null)
+            Func<HttpRequestMessage, Task> onIntercepted = null,
+            bool ignoreQueryString = false)
```

Comment:
If you look at #9, I've started to add new options as builder-only things. In an earlier iteration of that PR I started to add new optional parameters here and it got...messy.

Reply:
Ah OK I was just merrily adding things anywhere it looked like it made sense.  Will remove.